tests/app_net_ssa_test.py

```diff
@@ -28,13 +30,20 @@
 _XdsTestServer = xds_k8s_testcase.XdsTestServer
 _XdsTestClient = xds_k8s_testcase.XdsTestClient
 
-_REPLICA_COUNT = 3
+# Type aliases.
+_Lang: TypeAlias = skips.Lang
 
-# This is here temporarily to run this test separately from other app_net tests.
-# TODO(sergiitk): Move into app_net_test.py
+# Constants.
+_REPLICA_COUNT: Final[int] = 3
 
 
 class AppNetSsaTest(xds_k8s_testcase.AppNetXdsKubernetesTestCase):
+    @staticmethod
+    def is_supported(config: skips.TestConfig) -> bool:
```

Comment:
on which versions is this run right now?

Reply:
Now it's not restricted, but prod buildscripts only exist for cpp v1.62.x+. However, if we want to enable it in staging, we need this loging in the test itself.